app/src/main/java/org/wikipedia/readinglist/ReadingListsSurveyHelper.kt

```diff
@@ -0,0 +1,93 @@
+package org.wikipedia.readinglist
+
+import android.app.Activity
+import android.content.Context
+import android.widget.TextView
+import androidx.appcompat.app.AlertDialog
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.analytics.ReadingListsFunnel
+import org.wikipedia.page.LinkMovementMethodExt
+import org.wikipedia.settings.Prefs
+import org.wikipedia.util.CustomTabsUtil
+import org.wikipedia.util.GeoUtil
+import org.wikipedia.util.StringUtil
+import java.util.*
+
+object ReadingListsSurveyHelper {
+    private const val privacyPolicyUrl = "https://foundation.wikimedia.org/wiki/Legal:Wikipedia_Android_App_Onboarding_Survey_Privacy_Statement"
```

Comment:
could we please move this to strings_no_translate?

Reply:
yep!